src/usePopover.ts

```diff
@@ -25,10 +25,13 @@ export const usePopover = ({
   const positionPopover = useCallback<PositionPopover>(
     (
       positionIndex: number = 0,
-      childRect: ClientRect = childRef.current.getBoundingClientRect(),
+      childRect: ClientRect | undefined = childRef?.current?.getBoundingClientRect(),
       popoverRect: ClientRect = popoverRef.current.getBoundingClientRect(),
-      parentRect: ClientRect = containerParent.getBoundingClientRect(),
+      parentRect: ClientRect | undefined = containerParent?.getBoundingClientRect(),
     ) => {
+      if (!childRect || !parentRect) {
```

Comment:
Prevent errors when childRect or parentRect are undefined. This was the underlying error causing the `getBoundingClientRect` error.